utopia-remix/app/services/fgaService.server.ts

```diff
@@ -90,3 +70,72 @@ export async function canSeeLiveChanges(projectId: string, userId: string): Prom
 export async function canManageProject(projectId: string, userId: string): Promise<boolean> {
   return checkUserProjectPermission(projectId, userId, 'can_manage')
 }
+
+//
```

Comment:
Was this deliberate?

Reply:
just as a visual separation between related groups of functions (I can remove it though of course :) )

Reply:
Please